modules/matterhorn-capture-admin-service-impl/src/main/java/org/opencastproject/capture/admin/impl/CaptureAgentStateServiceImpl.java

```diff
@@ -505,35 +502,35 @@ private void updateAgentInDatabase(AgentImpl agent, boolean updateFromCache) {
     EntityManager em = null;
     EntityTransaction tx = null;
     try {
-      em = emf.createEntityManager();
-      tx = em.getTransaction();
-      tx.begin();
-      AgentImpl existing = getAgentEntity(agent.getName(), agent.getOrganization(), em);
-
+      //This is the cached last-heard-from time
+      Long cached = -1L;
```

Comment:
Could you use a more meaningful variable name and remove the comment?

E.g. lastHeardFromCached or something like that.